nyc_taxi/nyc_taxi-nongeo.ipynb

```diff
@@ -154,9 +145,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "shaded = hd.spread(hd.datashade(ranged, normalization='linear'))\n",
-    "labelled = shaded.redim.label(trip_distance=\"Distance, miles\", fare_amount=\"Fare, $\")\n",
-    "labelled"
+    "df.hvplot.scatter('trip_distance', 'fare_amount', rasterize=True, dynspread=True,\n",
```

Comment:
Same comment as above on `alpha` and on the fact that the plot looks quite different.